src/shared/watchdog.c

```diff
@@ -123,7 +125,7 @@ static int open_watchdog(void) {
                          ident.firmware_version,
                          fn);
 
-        return update_timeout();
+        return update_timeout(false);
```

Comment:
for boolean function params which are not obvious to understand when looking at them in calls like this, we try to add comments in style `/* foo= */` before them now, to make it easier to follow. i.e. please change this to:

```
return update_timeout(/* enable= */ false);
```

That said, is this change really worth it? It complicates our code needlessly? I mean, it doesn't hurt to enable it, even if it's already enabled?

Reply:
also, if this is kept (and I wouldn't, as mentined), it deserves a comment, i.e. why it's not necessary to enable the wacthdog device, and that that's part of the watchdog kernel API

Reply:
> for boolean function params which are not obvious to understand when looking at them in calls like this, we try to add comments in style `/* foo= */` before them now, to make it easier to follow. i.e. please change this to:

IMHO, it might make sense for function that takes 3 or more parameters but for those that takes only 1... but ok.